README-ContextRegistryService-soapui-project.xml

```diff
@@ -832,4 +832,4 @@
          <token>${#Project#token}</token>
       </ser:unregister>
    </soapenv:Body>
-</soapenv:Envelope>]]></con:request><con:credentials><con:authType>No Authorization</con:authType></con:credentials><con:jmsConfig JMSDeliveryMode="PERSISTENT"/><con:jmsPropertyConfig/><con:wsaConfig mustUnderstand="NONE" version="200508" action="http://services.rt.fs.documentum.emc.com/ContextRegistryServicePort/unregisterRequest"/><con:wsrmConfig version="1.2"/></con:call></con:operation></con:interface><con:properties><con:property><con:name>userName</con:name><con:value>some-user</con:value></con:property><con:property><con:name>password</con:name><con:value>some-password</con:value></con:property><con:property><con:name>token</con:name><con:value>some-token</con:value></con:property></con:properties><con:wssContainer/><con:oAuth2ProfileContainer/><con:oAuth1ProfileContainer/><con:sensitiveInformation/></con:soapui-project>
\ No newline at end of file
+</soapenv:Envelope>]]></con:request><con:credentials><con:authType>No Authorization</con:authType></con:credentials><con:jmsConfig JMSDeliveryMode="PERSISTENT"/><con:jmsPropertyConfig/><con:wsaConfig mustUnderstand="NONE" version="200508" action="http://services.rt.fs.documentum.emc.com/ContextRegistryServicePort/unregisterRequest"/><con:wsrmConfig version="1.2"/></con:call></con:operation></con:interface><con:properties><con:property><con:name>userName</con:name><con:value>xhibit_user</con:value></con:property><con:property><con:name>password</con:name><con:value>Password@1</con:value></con:property><con:property><con:name>token</con:name><con:value>some-token</con:value></con:property></con:properties><con:wssContainer/><con:oAuth2ProfileContainer/><con:oAuth1ProfileContainer/><con:sensitiveInformation/></con:soapui-project>
```

Comment:
We need to be careful with the userName / password properties here.
If having them in Project Properties is an issue, we should change them to be Global Properties.
It looks like you can override them under "Preferences"

Reply:
I'm going to remove the properties at the project level and document the keys. I assumed that the keys/values would not be exported back to the main repo.

Reply:
Done